src/validateInput.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+function validateInput(inputNumber) {
+  const validationResult = [];
+
+  const isFourDigits = inputNumber.length === 4;
```

Comment:
Replace the hardcoded value 4 with a constant or a configurable variable, e.g., const NUMBER_LENGTH = 4.